Snowflake.Data/Core/FileTransfer/SFFileTransferAgent.cs

```diff
@@ -132,6 +132,11 @@ class SFFileTransferAgent
         private string streamDestFileName = null;
         private string destStagePath = null;
 
+        /// <summary>
+        /// Placeholder threshold value.
+        /// </summary>
+        private long DATA_SIZE_THRESHOLD = 9223372036854775807;
```

Comment:
what's special about this number? Can you put more comments for this specific value?

Reply:
The number is equivalent to max long value. I'll edit it to use C#'s max long instead for clarity